.circleci/config.yml

```diff
@@ -1,11 +1,12 @@
 version: 2
 
 jobs:
-  build:
-    working_directory: /home/reaction
+  # run automated tests
+  tests:
+    working_directory: ~/reaction
```

Comment:
Changed from `/home/reaction` to `~/reaction`. The `8.9.3-browsers` image throws permission denied error on root folder. Any better way?

Reply:
Hi @impactmass. This is a minor suggestion because I'm sure that this will work as-is. `~` is expanded to home by the shell when it's not prefixed. But `~` is a typing convenience for the command line. In scripts it's generally preferred to use the `$HOME` var.

See:

* [SO: Difference between home and tilde](https://stackoverflow.com/questions/11587343/difference-between-home-and-tilde)
* [SO: Why use home over tilde in a shell script](https://stackoverflow.com/questions/5930671/why-use-home-over-tilde-in-a-shell-script)
* [Bash Tilde Expansion Documentation](https://www.gnu.org/software/bash/manual/html_node/Tilde-Expansion.html).

---

That said, you did the right thing. The old path didn't work on that image because the user `reaction` doesn't exist in `circleci/node:8.9.3-browsers` and therefore the `/home/reaction` directory doesn't exist either.

Reply:
Thanks. I changed to $HOME. With that circleci fails on the checkout step. 
`Unable to create directory '$HOME/reaction`. Not sure why yet. 

Also, I did not get this part `The old path didn't work on that image because the user `reaction` doesn't exist ... and therefore the /home/reaction`. Were we creating a `reaction` user in the previous setup?

Reply:
There was no `reaction` user.  That was just the directory I picked to checkout the code within the image.  `~/` definitely was a problem previously though, so that's why I explicitly set it to `/home/`.